Sources/TuistGenerator/Models/InfoPlist.swift

```diff
@@ -2,30 +2,139 @@ import Basic
 import Foundation
 import TuistCore
 
-public enum InfoPlist: Equatable, ExpressibleByStringLiteral, ExpressibleByUnicodeScalarLiteral, ExpressibleByExtendedGraphemeClusterLiteral {
+public enum InfoPlist: Equatable {
+    public indirect enum Value: Decodable, Equatable {
+        case string(String)
+        case integer(Int)
+        case boolean(Bool)
+        case dictionary([String: Value])
+        case array([Value])
+
+        var value: Any {
+            switch self {
+            case let .array(array):
+                return array.map { $0.value }
+            case let .boolean(boolean):
+                return boolean
+            case let .dictionary(dictionary):
+                return dictionary.mapValues { $0.value }
+            case let .integer(integer):
+                return integer
+            case let .string(string):
+                return string
+            }
+        }
+
+        public init(from decoder: Decoder) throws {
```

Comment:
It doesn't look like this is used (the model loader maps directly to this type, it doesn't go via the decode path)

Reply:
I'll remove it 👍